utils/resultwriter.go

```diff
@@ -25,11 +28,23 @@ import (
 )
 
 const (
-	BaseDocumentationURL = "https://docs.jfrog-applications.jfrog.io/jfrog-security-features/"
+	BaseDocumentationURL           = "https://docs.jfrog-applications.jfrog.io/jfrog-security-features/"
+	CurrentWorkflowNameEnvVar      = "GITHUB_WORKFLOW"
+	CurrentWorkflowRunNumberEnvVar = "GITHUB_RUN_NUMBER"
+
+	MissingCveScore = "0"
+	maxPossibleCve  = 10.0
+
+	// #nosec G101 -- Not credentials.
+	patchedBinarySecretScannerToolName = "JFrog Binary Secrets Scanner"
+	jfrogFingerprintAlgorithmName      = "jfrogFingerprintHash"
 )
 
-const MissingCveScore = "0"
-const maxPossibleCve = 10.0
+var (
```

Comment:
why var and not the prevous const? are they changing?

Reply:
they just moved to a centered definition of costs